spec/03-plugins/37-opentelemetry/07-utils_spec.lua

```diff
@@ -0,0 +1,52 @@
+local LOG_PHASE = require("kong.pdk.private.phases").phases.log
+
```

Comment:
```suggestion
require "spec.helpers"

```

we still need to require helpers because it does some needed setup on the environment, but don't assign it to a local variable, or else the linter will complain because it's unused.